### ISSUE TITLE
[Enhancement] Support horizontally centered dropdown

### ISSUE THREAD

**Issue by askarby** [kirbydesign/designsystem#1772]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When placing a `kirby-dropdown` component not aligned "to the left" or "to the right" of the screen (on a smaller device, in this example a iPhone X) the dropdown cannot be fully rendered.

It would be great to have a dropdown strategy where the options are not clipped, even though the dropdown is placed horizontally. 

## Describe how to reproduce the bug

1. Place a `kirby-dropdown` horizontally centered on the screen
2. Emulate a iPhone X (narrow the window accordingly)
3. Trigger the drop-down to display
4. See that it won't display fully on the screen!

## Which Kirby version was used?
3.0.15

## What was the expected behavior?
The drop-down will display (fully) on the screen - perhaps centered?

## Add any screenshots
![image](https://user-images.githubusercontent.com/44014/134872941-c27bfda2-3478-4a13-9788-f672c2b0fe19.png)
![image](https://user-images.githubusercontent.com/44014/134872956-5c8dbb8a-3f11-4fed-b29f-010bcd4e754f.png)

## Please complete the following information:
- OS: MacOS
- Browser Chrome
- Browser version 93

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](../SUPPORT.md/#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
I believe this is a duplicate of #1683. We are currently implementing new 'Popover' functionality that should hopefully mitigate this. You can follow along in #1671.

**Comment by RasmusKjeldgaard** [comment]
Reopening this, as there were more nuances to it than I first thought. 
I have removed the parts of this issue that duplicated #1683, but there is still a need for a strategy for placing dropdowns beside each other, or horizontally centered. E.g. situations where one or more dropdowns are not right or left aligned. 
I have updated the issue description accordingly.

**Comment by RasmusKjeldgaard** [comment]
![Screenshot_20211004-125055](https://user-images.githubusercontent.com/42470636/135839195-a6eae228-2c1d-4861-99fe-2192cf5bc6ac.jpg)
This is also a problem when the app is running on a device. So not only related to specific device or browser.

**Comment by henrikvoetmand** [comment]
**UX refinement:**
Define max width of popover layer (425px?)
When viewport is smaller, set the popover width to 100% (of the container)
No matter the dropdown button horizontal position.
When viewport is larger than max width - have the option to left, right or center the horizontal, position (relative to the dropdown btn)

![Popover - alignment](https://user-images.githubusercontent.com/71710012/155142183-7f705307-f85c-46a3-a7ce-84599983950c.png)

**Comment by MadsBuchmann** [comment]
Before we do this - we should consider whether we should use the popover functionality introduced in #1671. This requires also solving #1793. Perhaps by always displaying the scrollbar? (see [this tech refinement](https://github.com/kirbydesign/designsystem/issues/1543#issuecomment-851959447) from #1543).

**Comment by RasmusKjeldgaard** [comment]
Consider looking at this when solving #3736

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!

**Comment by henrikvoetmand** [comment]
This issue is still relevant.
